src/main/cpp/sim/run_stride.cpp

```diff
@@ -92,9 +93,14 @@ void StrideSimulatorResult::AfterSimulatorStep(
 		cp->CloseFile();
 	}
 	*/
+	auto& pop = sim.GetPopulation();
 	run_clock.Stop();
 	auto infected_count = sim.GetPopulation()->get_infected_count();
 	cases.push_back(infected_count);
+
+	if(generate_vis_data && pop.has_atlas)
```

Comment:
The spacing in this `if` statement looks a little off. Can you `clang-format` this file?